src/keycloak/chart/templates/istio-admin.yaml

```diff
@@ -19,7 +19,6 @@ spec:
               - "8080"
             paths:
               - "/admin*"
-              - "/realms/master*"
```

Comment:
Was this removal intentional? I assume we'd still want to only expose the master realm via the admin gateway?